Author: Apollon Oikonomopoulos <apoikos@debian.org>
Description: Fix libnss_*.so lookup
 Rely on dlopen()'s default search paths to find libnss_*.so, rather than
 hardcode its absolute path. Since libnss modules live under multiarch triplet
 paths in Debian, this will do the right thing.
Forwarded: no
Last-Update: 2016-02-19
 
--- a/src/auth/userdb-nss.c
+++ b/src/auth/userdb-nss.c
@@ -86,7 +86,7 @@
 	const char *name = module->nss_module.name;
 	char *path;
 
-	path = p_strdup_printf(pool, "/usr/lib/libnss_%s.so", name);
+	path = p_strdup_printf(pool, "libnss_%s.so", name);
 	module->nss_module.handle = dlopen(path, RTLD_GLOBAL | RTLD_NOW);
 	if (module->nss_module.handle == NULL)
 		i_fatal("dlopen(%s) failed: %s", path, dlerror());
